src/App.js

```diff
@@ -125,14 +130,23 @@ function App() {
                 deletePokemon={deletePokemon}
             />
 
-            <AddPokemon setPokemonCardData={setPokemonCardData} save={save} />
+            <AddPokemon
+                setPokemonCardData={setPokemonCardData}
+                pokemonTypes={pokemonTypes}
+                pokemonCardSets={pokemonCardSets}
+                save={save}
+                updateCardData={updateCardData}
+            />
 
             {Object.keys(selectedPokemon).length !== 0 && (
                 <EditPokemon
                     setPokemonCardData={setPokemonCardData}
```

Comment:
you can remove this prop now that you are using updateCardData and setPokemonCardData is no longet being used in this component